### ISSUE TITLE
Reduce number of process for cros device moves

### ISSUE THREAD

**Issue by dmd408** [GAM-team/GAM#1425]
On one hand, it's awesome that GAM splits a bulk cros device moves into 25 processes and flies through them..... up to the back off limits anyhow.

The default quota limit for cros device moves is 60 per minute.  The 25 concurrent processes easily exceeds that limit.  For large moves, it would be faster if there were fewer processes and no slow downs.

So why not a number that works out better with the moves per second relative to the default quota limit provided by Google?  I could request a quota limit increase, but a lot of people won't even know how to do that.  And it would have to be a pretty high quota to make sure it's not hit still with how fast 25 processes moves thing.

The coolest of cool would be if I could decide on the number of processes to run concurrently.  But I'd definitely be happy with fewer processes, thus a slower sprint, but a faster longer race.  And it really doesn't take many before hitting this issue.

TL;DR:  25 processes is great for moving a small number of devices.  It's super fast.  But if you need to move a fair number of devices, the back offs from exceeding the quota limits make it slower in the long run than if there were fewer processes and fewer back offs in turn.

**Comment by taers232c** [comment]
It would be useful to see the command you executed.

**Comment by dmd408** [comment]
It's just a straightforward and simple bulk cros ou update

gam csv [file path] gam cros update cros ~[heard for deviceID] ou ~[header for OU to move device to]

**Comment by taers232c** [comment]
Contact me directly via Meet/Zoom.

ross.scroggs @ gmail.com
